File: exercise-03-printing-quotes/PrintingQuotes.java
class PrintingQuotes {
  public static void main(String[] args) {
    var quote = System.console().readLine("What is the quote? ");
    var author = System.console().readLine("Who said it? ");
    System.out.println(author + " said \"" + quote + "\"");
  }
}

File: 03-printing-quotes/printing-quotes.js
const readline = require('readline');
const rl = readline.createInterface(process.stdin, process.stdout);

rl.question('What is the quote? ', (quote) => {
  rl.question('Who said it? ', (author) => {
    console.log(`${author} said "${quote}"`);
    rl.close();
  });
});
